src/Order/Domain/ValueObject/OrderItem/TotalPrice.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace App\Order\Domain\ValueObject\OrderItem;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Embeddable()
+ */
+class TotalPrice
+{
+    /**
+     * @ORM\Column(name="total_price", type="float")
+     */
+    private float $price;
+
+    public function __construct(float $price)
```

Comment:
could we merge price and total price and dish price as a single object? Why did you split it into different objects? What different rules do they have?